specs/bls_signature.md

```diff
@@ -6,23 +6,25 @@
 
 ## Table of contents
 <!-- TOC -->
```

Comment:
is this `TOC` comment still needed?

Reply:
not for `doctoc`, but it looks like some text editors adds such a comment:https://github.com/nok/markdown-toc https://github.com/AlanWalk/Markdown-TOC